modules/light-clients/08-wasm/types/client_state_test.go

```diff
@@ -26,6 +26,10 @@ import (
 	ibcmock "github.com/cosmos/ibc-go/v8/testing/mock"
 )
 
+var (
+	MockClientStateBz = []byte("client-state-data")
```

Comment:
thank! Best to be moved in `wasm_endpoint.go` with a couple of the other mock values. In addition, this can be used in a couple of other places, a grep for `clientstatebz` should find many of these spots.

Reply:
Made the shift, I will limit this PR to the mock values needed just within 08-wasm, noticed other places as well, will create new issues and raise more PRs